iroh-relay/src/lib.rs

```diff
@@ -0,0 +1,30 @@
+//! Package `relay` implements a revised version of the Designated Encrypted Relay for Packets
+//! (DERP) protocol written by Tailscale.
```

Comment:
This is now the first thing that people see when they open the docs for the relay package.

Given that this is way more prominent now, perhaps it's worth adjusting this to the wording used in the README, which personally I found more fitting?